pages/integrations/output_framework.rst

```diff
@@ -0,0 +1,146 @@
+.. _output_framework:
+
+***************************
+Enterprise Output Framework
+***************************
+
+The Enterprise Output Framework provides the ability to forward data from your Graylog 
+cluster to external systems using a variety of network transport methods and payload 
+formats. In addition, you can configure Framework-based Outputs to use 
+:doc:`Processing Pipelines <../pipelines/pipelines>` to filter, modify, and enrich the 
+outbound messages.
+
+.. note:: This is an Enterprise Integrations feature and is only available since Graylog 
+          version 3.3.3, thus an Enterprise license is required. See the 
+          :doc:`Integrations Setup <setup>` page for more info.
+
+Enterprise Framework Outputs
+----------------------------
+
+The Enterprise Output Framework provides a number of new Outputs for the various network 
+transport types. All of these Outputs first write messages to an on-disk journal in the 
+Graylog cluster.  Messages stay in the on-disk journal until the Output is able to 
+successfully send the data to the external receiver.
+
+Once the messages have been written to the journal, they are optionally run through a 
+processing pipeline to modify or enrich logs with additional data, transform the message
+contents, or filter out any some logs before sending.
```

Comment:
For clarification - the data is written to the journal first, then send to the processing pipeline for modification and send to the selected output. 

Or is the output modified by the processing pipeline, written to the journal, read in and modified to the desired output format and send to the target?

Reply:
The flow goes like this:

1. Outgoing message is written to the journal
1. Journal handler executes optional pipeline
1. Journal handler formats message
1. Journal handler sends formatted message
1. Journal handler updates journal to indicate that the message is done being handled

Reply:
adding a diagram of the flow of the data might assist in any confusion a user might have, similar to the way docs for the [forwarder](https://docs.graylog.org/en/3.3/pages/integrations/forwarder.html) are.

Reply:
Updated some styling and removed images on latest push.  Jan will work on the diagram and do some additional styling work